src/level_files_manager.rs

```diff
@@ -81,8 +79,7 @@ pub fn level_files_manager_section(
             construction_specs,
             yoleck_managed_query,
             keep_levels_query,
-            editor_state,
-            mut next_editor_state,
+            _editor_state,
```

Comment:
I see you do use it. Is the leading underscore a leftover from some attempted refactor?